daemon/mgr/container.go

```diff
@@ -49,9 +49,6 @@ import (
 type ContainerMgr interface {
 	// 1. the following functions are related to regular container management
 
-	// Restore containers from meta store to memory and recover those container.
-	Restore(ctx context.Context) error
```

Comment:
I do not think we should remove this part. Restore action is one of the essential functionality of ContainerManager. I think maybe your removing this is because the adding restore action in `NewContainerManager `. While I will add my reason in the following comments.